rust/src/writer/mod.rs

```diff
@@ -54,12 +54,12 @@ pub(crate) enum DeltaWriterError {
         sample_error: ParquetError,
     },
 
-    // TODO: derive Debug for Stats in delta-rs
     /// Serialization of delta log statistics failed.
     #[error("Serialization of delta log statistics failed")]
     StatsSerializationFailed {
         /// The stats object that failed serialization.
-        stats: Stats,
+        #[from]
+        stats: serde_json::Error,
```

Comment:
This is the only manual change. Clippy was complaining that the error variant was much larger then all the others. I felt forwarding the error rather then the actual data that caused the error is a reasonable change.

Reply:
That seems fair. The other option is to `Box` the Stats, but I think the `serde_json` error should be sufficient.

Could we update the doc comment above, though?

Reply:
Of course! Going forward we way consider lifting the docs requirement for error variants. I feel the description test and variant name are usually expressive enough.